### ISSUE TITLE
Bootstrap Tooltip / Popover Dismissal Causing HTML element to be hidden

### ISSUE THREAD

**Issue by jayb611** [twbs/bootstrap#19698]
Solution i did is this

Replace Bootstrap.js line 1404 as below

    //if (obj instanceof $.Event) {
        //  self.inState[obj.type == 'focusout' ? 'focus' : 'hover'] = false
        //}
To
 

    //09-04-2016 Changed condition if it is hover out then set instate as true
        //if (obj instanceof $.Event) {
        //  self.inState[obj.type == 'focusout' ? 'focus' : 'hover'] = false
        //}
        if (obj instanceof $.Event) {
            if (obj.type == 'focusout') {
                self.inState['focus'] = false;
            }
            else {
                self.inState['hover'] = true;
            }
        }

Basically it was setting "inState" to "false" so when hover out happened it was continuing on other code instead of returning. the continuing code was to set tooltip hide.

**Comment by cvrebert** [comment]
@jayb611 Please describe the problem in more detail, preferably with a JS Bin or JS Fiddle example.

**Comment by twbs-closer** [comment]
Hey there!

We're automatically closing this issue since the original poster (or another commenter) hasn't yet responded to the question or request made to them 14 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

(*Please note that this is an [automated](https://github.com/twbs/no-carrier) comment.*)
